corehq/apps/app_execution/forms.py

```diff
@@ -39,15 +40,25 @@ def __init__(self, *args, **kwargs):
         self.helper = hqcrispy.HQFormHelper()
 
         self.helper.layout = crispy.Layout(
-            "name",
-            "domain",
-            "app_id",
-            "username",
-            "workflow",
-            "sync_before_run",
-            "form_mode",
-            "run_every",
-            "notification_emails",
+            crispy.Div(
+                crispy.Div(
+                    "name",
+                    "domain",
+                    "app_id",
+                    "username",
+                    "sync_before_run",
+                    "form_mode",
+                    "run_every",
+                    "notification_emails",
+                    css_class="col",
+                ),
+                crispy.Div(
```

Comment:
If I understand the form correctly, and I am not sure I do, is the workflow field an alternative to the har import? This is not clear from this layout.

Reply:
The HAR file import is a separate input and button at the bottom of the form. Importing the file will submit the whole form, process the file and then return the user to the same page with all the existing data except the workflow is replaced by the one generated from the HAR file.

![image](https://github.com/dimagi/commcare-hq/assets/249606/79eb8973-963b-48a4-9509-640578c8aa8c)